src/main/java/net/minecraftforge/event/entity/player/PlayerInteractEvent.java

```diff
@@ -326,6 +327,42 @@ public LeftClickEmpty(EntityPlayer player, @Nonnull ItemStack stack)
     }
 
     /**
+     * This event is fired on the client side when a player middle clicks while aiming on a block or entity.
+     * There is no update packet for it, you have to send your own when needed.
+     * You can change the Result ItemStack here or set it to {@link ItemStack#EMPTY} to prevent further updating.
+     * Cancel the event if you want that nothing happens.
```

Comment:
I don't understand the difference between "prevent further updating" and canceling the event.